src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+  @Override
+  public void doWork() {
+    System.out.println("Excavator started its work");
+  }
+
+  @Override
+  public void stopWork() {
+    System.out.println("Excavator stopped working");
```

Comment:
Similarly to the `doWork` method, the message in `stopWork` should be more informative and indicate that an `Excavator` has stopped working.